doc/use_cli_in_airgapped_clouds.md

```diff
@@ -3,8 +3,15 @@
 ## Introduction
 While Azure CLI is not fully supported in an airgapped cloud which has no public internet connectivity, most commands can run without issues if the underlying service is supported in that cloud.
 
+Here is a list of known CLI features that are not supported in airgapped clouds:
+* Manage extensions with extension names. For instance, you can not install the `azure-devops` extension with `az extension add --name azure-devops`. You need to provide a local path to a wheel file or a URL for downloading the wheel file that is accessible in the airgapped cloud with `--source` option.
+* `az upgrade` to update to the latest CLI version. You can follow the below install instructions to update CLI.
```

Comment:
what's the different between upgrade and update? I suggest use same word.

Reply:
Quite similar. Some [website](https://rocketit.com/difference-between-update-upgrade/) says upgrade has bigger changes than update.